readme.md

```diff
@@ -182,23 +182,35 @@ input values, and does not produce side effects.
 const greet = (name) => 'Hi, ' + name
 
 greet('Brianne') // 'Hi, Brianne'
-
 ```
 
-As opposed to:
+As opposed to each of the following:
 
 ```js
-
-let greeting
+window.name = 'Brianne'
 
 const greet = () => {
-  greeting = 'Hi, ' + window.name
+  return 'Hi, ' + window.name
 }
 
 greet() // "Hi, Brianne"
+```
+
+The above example's output is based on data stored outside of the function...
 
+```js
+let greeting
+
+const greet = (name) => {
+  greeting = 'Hi, ' + name
```

Comment:
Isn't that kind of... Obtuse?  I feel like it really makes the code harder to grok in favor of reducing the line count.  Assuming someone wasn't super familiar with ES6 syntax, I think that would be really hard code to read.